frontend/src/containers/dashboard-container/index.js

```diff
@@ -60,8 +61,24 @@ export const DashboardContainer = () => {
   /* istanbul ignore next */
   const onDrop = useCallback(
     (_item, monitor) => {
+      const urls = monitor
+        .getItem()
+        .urls[0].split('https://')
+        .map(url => {
+          return 'https://' + url
+        })
+        .slice(1)
       if (monitor) {
```

Comment:
Hey, out of interest what does `monitor.getItem().urls[0]` return?

Reply:
Hey, it returns the urls of the tracks dragged as a single string all chained together. e.g. "https://open.spotify.com/track/60W7WuHF8u2GcIywPNeJDrhttps://open.spotify.com/track/6rlOTmO9UzSIKmTorPq3UXhttps://open.spotify.com/track/4vxm2gnST3bQer60vY1pJp"